gen3/cli/drs_pull.py

```diff
@@ -139,6 +140,12 @@ def download_object(
         delete_unpacked_packages,
     )
 
+    for drs_object_id in res:
+        if res[drs_object_id].status != "downloaded":
+            click.echo(f"Object {drs_object_id} have not been successfully downloaded.")
```

Comment:
```suggestion
            click.echo(f"Object {drs_object_id} has failed to downloaded.")
```

remove "success" b/c at a glance someone might miss the "not" and think it worked